Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityPointerProfile.cs

```diff
@@ -30,6 +30,23 @@ public class MixedRealityPointerProfile : BaseMixedRealityProfile
         /// </summary>
         public LayerMask[] PointingRaycastLayerMasks => pointingRaycastLayerMasks;
 
+        [SerializeField]
+        [Tooltip("The LayerMasks, in prioritized order, that are used to determine the touchable objects.")]
```

Comment:
Can you clarify that to make an object touchable it must have both nearinteractiontouchable AND the layer mask?

Reply:
Added \<remarks\> to both the PokePointer and BaseNearInteractionTouchable.